src/notifications/using_webhooks.md

```diff
@@ -0,0 +1,143 @@
+# Using Webhooks
+
+## Register a Webhook
+
+Once your vendor [NDS is set up](setup_nds.md) and can accept POST requests from the SDK services, you can within your mail application register the webhook URL with the Breez SDK by calling the register webhook API as follows:
```

Comment:
> ... you can within your mail application ...

Mail application?

Reply:
Was a typo, should be "...main application..."